node/src/components/consensus.rs

```diff
@@ -139,11 +139,11 @@ impl Debug for ConsensusMessage {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         match self {
             ConsensusMessage::Protocol { era_id, payload: _ } => {
-                write!(f, "Protocol {{ era_id.0: {}, .. }}", era_id.0)
+                write!(f, "Protocol {{ era_id: \"{}\", .. }}", era_id)
```

Comment:
Since this is `Debug`:
```suggestion
                write!(f, "Protocol {{ era_id: {:?}, .. }}", era_id)
```